views/default/tab_client_snapshots.pdt

```diff
@@ -46,16 +48,16 @@
                             </div>
                         </div>
 
-                        <a class="btn btn-xs btn-danger remove-snapshot" href="#" data-id="<?php $this->Html->_($snapshotid); ?>" data-toggle="modal" data-target="#remove-snapshot<?php echo $snapshotid; ?>">
+                        <a class="btn btn-xs btn-danger remove-snapshot" href="#" data-id="<?php $this->Html->_($snapshot->SNAPSHOTID); ?>" data-toggle="modal" data-target="#remove-snapshot<?php echo $snapshot->SNAPSHOTID; ?>">
```

Comment:
The $snapshot->SNAPSHOTID should be HTML-safe like in other locations

Reply:
It is. Html::_() does that.